dotcom-rendering/src/server/prod-server.ts

```diff
@@ -21,25 +20,20 @@ import { handleFront, handleFrontJson } from '../web/server/index.front.web';
 import { recordBaselineCloudWatchMetrics } from './lib/aws/metrics-baseline';
 import { getContentFromURLMiddleware } from './lib/get-content-from-url';
 import { logger } from './lib/logging';
-import { requestLoggerMiddleware } from './lib/logging-middleware';
+import {
+	loggingStoreMiddleware,
+	requestLoggerMiddleware,
+	timedExpressJsonMiddleware,
+} from './lib/logging-middleware';
 import { recordError } from './lib/logging-store';
 
-// Middleware to track route performance using 'response-time' lib
-// Usage: app.post('/Article', logRenderTime, renderArticle);
-const logRenderTime = responseTime(
-	(_1: Request, _2: Response, renderTime: number) => {
-		logger.info('Page render time', {
-			renderTime,
-		});
-	},
-);
-
 export const prodServer = (): void => {
 	logger.info('dotcom-rendering is GO.');
 
 	const app = express();
 
-	app.use(express.json({ limit: '50mb' }));
+	app.use(loggingStoreMiddleware);
```

Comment:
I've seperated some of the behaviour of `requestLoggerMiddleware` into a seperate middleware.

- loggingStoreMiddleware needs to be the first middleware to run in order to setup the logging store before `timedExpressJsonMiddleware`
- loggingStoreMiddleware needs to be the first middleware to run in order to get an accurate reading on the total request time
- requestLoggerMiddleware needs access to the parsed request body meaning it has to run after the `timedExpressJsonMiddleware`